<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SavedDialog</class>
 <widget class="QDialog" name="SavedDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fingerprint saved</string>
  </property>
  <property name="windowIcon">
   <iconset resource="widget-res.qrc">
    <normaloff>:/new/prefix1/res/gnome-user-interface.png</normaloff>:/new/prefix1/res/gnome-user-interface.png</iconset>
  </property>
  <layout class="QVBoxLayout" stretch="4,2">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="modeLabel">
         <property name="text">
          <string>Fingerprint was saved to disk.</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Select another finger?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="margin">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="yesButton">
       <property name="text">
        <string>&amp;Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="noButton">
       <property name="text">
        <string>&amp;No</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="widget-res.qrc"/>
  <include location="widget-res.qrc"/>
  <include location="widget-res.qrc"/>
  <include location="widget-res.qrc"/>
 </resources>
 <connections/>
</ui>
